### ISSUE TITLE
GuiUnattended and Timezones improvement idea

### ISSUE THREAD

**Issue by hartsock** [vmware/pyvmomi#162]
Consider:

``` Python
gui_unattended = vim.vm.customization.GuiUnattended()
gui_unattended.timeZone = 35
gui_unattended.autoLogon = False
```

Shouldn't there be a way to set the timezones that doesn't involve using raw numbers?

**Comment by activars** [comment]
@hartsock 
Is there an example of setting timezone for a windows VM to UTC  (GMT without daylight saving)?

**Comment by hartsock** [comment]
@activars not to my knowledge, the documentation here:
 [docs/vim/vm/customization/GuiUnattended.rst](https://github.com/vmware/pyvmomi/blob/master/docs/vim/vm/customization/GuiUnattended.rst)

Seems to indicate that these are numbers derived from the guest OS. In this case we would look at the numbers described by [sysprep](https://technet.microsoft.com/en-us/library/bb457073.aspx). 

I don't usually do windows so I'm not familiar with that guest. 

On the vSphere end the `timezone` field *has* to be an `int` and that would have to map to one of these values [in sysprep](https://technet.microsoft.com/en-us/library/cc749073\(v=ws.10\).aspx) and it would seem that the windows sysprep tool doesn't understand UTC. It does understand GMT and GMT is _supposed_ to not observe daylight savings time. The British use the BST time code in the summer if memory serves ... (that means while Greenwich the city might or might not move it's clocks ... because they are a British city ... depending on how the Windows developers interpreted the timezone request you could argue it would be wrong to automatically move the clock of the guest. Or, the windows developers thought they might be helpful to residents of Greenwich by changing their clocks for them.)

**tl;dr**

 I don't know of any sample, and this is very specific to the guest.  Someone else with more experience with Windows guests might know better.

**Comment by activars** [comment]
Thank you for the quick reply.
I didn't found any other options after research. The final solution was to configure the timezone post-build after customization by using the `tzutil /s 'UTC'` command.
